<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="login_lb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 200); font-size:28pt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>4</number>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="email_lb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="indent">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_tb">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>321</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_tb">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>321</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="password_lb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="indent">
    <number>6</number>
   </property>
  </widget>
  <widget class="QPushButton" name="login_bt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 100); font-size:14pt; color:rgb(100, 255, 100)</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="create_account_bt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 80); font-size:12pt;  color:rgb(255, 255, 150)</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nickname_tb">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>321</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nickname_lb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Nickname</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="bg2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Assets/background.png);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Assets/bg2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>bg2</zorder>
  <zorder>login_lb</zorder>
  <zorder>email_lb</zorder>
  <zorder>email_tb</zorder>
  <zorder>password_tb</zorder>
  <zorder>password_lb</zorder>
  <zorder>login_bt</zorder>
  <zorder>create_account_bt</zorder>
  <zorder>nickname_tb</zorder>
  <zorder>nickname_lb</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
